repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl

```diff
@@ -112,7 +112,7 @@
             <h2><strong><@fmt.message key="headers.categories"/></strong></h2>
 
             <#list indicator.taxonomyList?keys as key>
-                <p class="filter-list__item__link" data-uipath="ps.indicator.taxonomy-${key}"><a title="Search for ${indicator.taxonomyList[key]}" href="../search/category/${key}">${indicator.taxonomyList[key]}</a></p>
+                <p class="filter-list__item__link" data-uipath="ps.indicator.taxonomy-${key}"><a title="Search for ${indicator.taxonomyList[key]}" href="/site/search/category/${key}/">${indicator.taxonomyList[key]}</a></p>
```

Comment:
This won't work on any environment except local. local URL has `/site`, others are just root

Reply:
so revert? It wasn't working for me how it was before

Reply:
I think there's a searchLink ftl variable in imports.ftl that you can use.

Reply:
Sorted - thanks!